src/structures/Node.ts

```diff
@@ -58,6 +58,11 @@ function check(options: NodeOptions) {
     typeof options.retryDelay !== "number"
   )
     throw new TypeError('Node option "retryDelay" must be a number.');
+  
+  if (typeof options.region !== "undefined" && 
+      ((typeof options.region !== "string" || 
+        !/.+/.test(options.region)) && (!Array.isArray(options.region) || options.region.length === 0 || !/.+/.test(options.region[0]))))
+    throw new TypeError('Node option "region" must be a non-empty string or array.');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585535)
```suggestion
    throw new TypeError("Node option \"region\" must be a non-empty string or array.");
```